src/definitions/elements/emoji.less

```diff
@@ -0,0 +1,67 @@
+/*!
+ * # Fomantic UI - Emoji
+ * http://github.com/fomantic/fomantic-ui/
+ *
+ *
+ * Released under the MIT license
+ * http://opensource.org/licenses/MIT
```

Comment:
Add link to license in repo
`https://github.com/fomantic/Fomantic-UI/blob/master/LICENSE.md`